lib/utils/upload.ts

```diff
@@ -2,6 +2,8 @@ import type { AxiosProgressEvent, AxiosResponse } from 'axios'
 import { generateRemoteUrl } from '@nextcloud/router'
 import { getCurrentUser } from '@nextcloud/auth'
 import axios from '@nextcloud/axios'
+import axiosRetry from 'axios-retry';
+axiosRetry(axios, { retries: 0, retryDelay: axiosRetry.exponentialDelay });
```

Comment:
I'm newbie in all this libraries :smile: . Should we do this configuration on `@nextcloud/axios` level?

Reply:
It should probably be done for the upload attempt only

Reply:
Yes it should be only be done for upload requests.